src/style.css

```diff
@@ -0,0 +1,134 @@
+/* #region Normalize */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --blue-color: #00acdc;
+  --text-dark-color: #060b35;
+  --text-light-color: #616070;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+/* #endregion */
+
+/* #region reset */
+body,
+p {
+  margin: 0;
+  padding: 0;
+}
+/* #endregion */
+
+/* #region card */
+.card {
```

Comment:
this particularly applies to it — the checklist, p. 11
let's split this styles into dedicated files